Shoot/README.md

```diff
@@ -1,38 +1,40 @@
 ## Shoot'nShare backend
-A simple example to demo OAuth2 authorization code grant with Keycloak.
 
+A simple example to demo OAuth2 authorization code grant with Keycloak.
 
 ### Pre-requisites
 
-* use [Keycloak Appliance Distribution (version 1.0.4.Final)](http://docs.jboss.org/keycloak/docs/1.0.4.Final/userguide/html/server-installation.html#Appliance_install) comes with a preconfigured Keycloak server (based on Wildfly). 
+Shoot'nShare backend uses [Keycloak](http://keycloak.jboss.org/) to authenticate or client apps, so we need deploy and configure the [Keycloak](http://keycloak.jboss.org/)
```

Comment:
/to authenticate or client apps,/to authenticate our client apps,/
/so we need deploy and configure the [Keycloak](http://keycloak.jboss.org/)/so we need to deploy and configure the [Keycloak](http://keycloak.jboss.org/) server./